src/VolumeRenderImpl.ts

```diff
@@ -19,7 +19,7 @@ export interface VolumeRenderImpl {
   doRender: (
     renderer: WebGLRenderer,
     camera: PerspectiveCamera | OrthographicCamera,
-    depthTexture?: DepthTexture | Texture
+    depthTexture?: DepthTexture | Texture | null
```

Comment:
null and undefined are both possible values for all these?

Reply:
three.js now represents the absence of a depth texture on a render target as `null`, but passing in depth textures is now also part of the public interface used by the simularium volumes implementation, and since leaving this property entirely missing has the same effect it makes sense to keep that option in there.